DataFormats/ProtonReco/src/classes.h

```diff
@@ -0,0 +1,22 @@
+#include "DataFormats/ProtonReco/interface/ForwardProton.h"
+#include "DataFormats/ProtonReco/interface/ForwardProtonFwd.h"
+
+#include "DataFormats/Common/interface/Ptr.h"
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/Common/interface/Wrapper.h"
+
+#include <vector>
+#include <set>
+
+namespace DataFormats_ProtonReco
+{
```

Comment:
these are not necessary anymore.
Only headers are needed.

Reply:
Namespace and struct blocks removed.